components/Table/Table.js

```diff
@@ -271,6 +274,33 @@ export class Table {
     }
   }
 
+  renderModalWindow(matchObj, rowId) {
+    const modalWindowContainer = document.createElement('div');
+    modalWindowContainer.classList.add('modal-window-container');
+    const bgShadow = document.createElement('div');
+    bgShadow.classList.add('bg-shadow');
+    this.tableContainer.append(modalWindowContainer, bgShadow);
+    bgShadow.addEventListener('click', () => {
+      modalWindowContainer.remove();
+      bgShadow.remove();
+    });
+    const modalWindow = new Modal(modalWindowContainer);
+    modalWindow.createModal(matchObj, rowId);
+  }
+
+  putDataToModal() {
```

Comment:
This doesn't look like put data to modal
I see that you add here listener and just call modal
Rename it more appropriate name, please